linkis-engineconn-plugins/engineconn-plugins/sqoop/src/main/scala/org/apache/linkis/engineconnplugin/sqoop/executor/SqoopOnceCodeExecutor.scala

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2019 WeBank
```

Comment:
Linkis has been already donated to Apache,
This is not a stand ASF header, `Copyright xxx` need to be removed

Reply:
ok